Antidote/CallViewController.m

```diff
@@ -0,0 +1,184 @@
+//
+//  CallViewController.m
+//  Antidote
+//
+//  Created by Chuong Vu on 7/1/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import "CallViewController.h"
+#import "Masonry.h"
+#import "Helper.h"
+#import "OCTCall.h"
+#import "OCTChat.h"
+#import "OCTSubmanagerCalls.h"
+
+NSString *const kCallViewControllerUserIdentifier = @"callViewController";
+
+@interface CallViewController () <RBQFetchedResultsControllerDelegate>
+
+@property (strong, nonatomic) OCTSubmanagerCalls *manager;
+@property (strong, nonatomic, readwrite) OCTChat *chat;
+@property (strong, nonatomic) OCTCall *call;
+@property (strong, nonatomic) RBQFetchedResultsController *callController;
+
+@property (strong, nonatomic) UILabel *nameLabel;
+@property (strong, nonatomic) UIButton *cancelCallButton;
+@property (strong, nonatomic) UILabel *timerLabel;
+
+@end
+
+@implementation CallViewController
+
+#pragma mark - Life cycle
+
+- (instancetype)initWithChat:(OCTChat *)chat submanagerCalls:(OCTSubmanagerCalls *)manager
+{
+    self = [super init];
+    if (! self) {
+        return nil;
+    }
+
+    _chat = chat;
+
+    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"chat == %@", chat];
```

Comment:
For some reason RBQFetchedResultsController doesn't get updated with such predicate. In ChatViewController I had to do `chat.uniqueIdentifier == %@", ...`